docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: "3"
+
+services:
+  tba:
+    # image: ghcr.io/the-blue-alliance/the-blue-alliance/tba-py3-dev:latest
+    build:
+      context: ./
+      dockerfile: ./ops/dev/docker/Dockerfile
```

Comment:
Using this `image` is the end goal, but for now this forces the local image build, for development/testing of this actual PR.